docs/guide/roles-and-multi-sig.md

```diff
@@ -0,0 +1,261 @@
+
+Remember that `rich`'s address was used on `basecoin init` and is included in the `--members` list. The command above will prompt for a password (which can also be piped into the command if desired) then - if executed correctly - return some data:
+
+```
+{
+  "check_tx": {
+    "code": 0,
+    "data": "",
+    "log": ""
+  },
+  "deliver_tx": {
+    "code": 0,
+    "data": "",
+    "log": ""
+  },
+  "hash": "4849DA762E19CE599460B9882DD42C7F19655DC1",
+  "height": 321
+}
+```
+showing the block height at which the transaction was committed and its hash. A quick review of what we did: 1) created a role, essentially an account, that requires a minimum of two (2) signatures from three (3) accounts (members). And since it was the account named `rich`'s first transaction, the sequence was set to 1.
+
+Let's look at the balance of the role that we've created:
+
+```
+basecli query account role:"10CAFE4E"
+```
+
+and it should be empty:
+
+```
+ERROR: Account bytes are empty for address role:10CAFE4E
+```
+
+Next, we want to send coins _to_ that role. Notice that because this is the second transaction being sent by rich, we need to increase `--sequence` to `2`: 
+
+```
+basecli tx send --fee=90mycoin --amount=10000mycoin --to=role:"10CAFE4E" --sequence=2 --name=rich
+```
+
+We need to pay a transaction fee to the validators, in this case 90 `mycoin` to send 10000 `mycoin` Notice that for the `--to` flag, to specify that we are sending to a role instead of an account, the `role:` prefix is added before the role. Because it's `rich`'s second transaction, we've incremented the sequence. The output will be nearly identical to the output from `create-role` above.
+
+Now the role has coins (think of it like a bank). 
+
+Double check with:
+
+```
+basecli query account role:"10CAFE4E"
+```
+
+and this time you'll see the coins in the role's account:
+
+```
+{
+  "height": 2453,
+  "data": {
+    "coins": [
+      {
+        "denom": "mycoin",
+        "amount": 10000
+      }
+    ],
+    "credit": []
+  }
+}
+```
+
+`Poor` decides to initiate a multi-sig transaction to himself from the role's account. First, it must be prepared like so:
+
+```
+basecli tx send --amount=6000mycoin --from=role:"10CAFE4E" --to=65D406E028319289A0706E294F3B764F44EBA3CF --sequence=1 --assume-role="10CAFE4E" --name=poor --multi --prepare=tx.json
+```
+
+you'll be prompted for `poor`'s password and there won't be any `stdout` to the terminal. Note that the address in the `--to` flag matches the address of `poor`'s account from the beginning of the tutorial. The main output is the `tx.json` file that has just been created. In the above command, the `--assume-role` flag is used to (not clear, since we have --from), while the `--multi` flag is used in combination with `--prepare`, to specify the file that is prepared for a multi-sig transaction.
```

Comment:
--from --to and --amount define the movement of money

you must explicitly say that you want to assume the role, rather than just signing it.  imagine you are a member of a 1 or 10 account, would every tx you sign automatically have access to that other account?

the way the code works, the tx must explicitly state you are going to try to assume this role to evalate your permissions for this one transaction.  does that make sense? can you explain it better?